scripts/ci/check_e2e_tarball.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+CI_SCRIPTS_DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
```

Comment:
The contents of this scripts were taken as is from the existing [travis-ci.sh script](https://github.com/vmware-tanzu/sonobuoy/blob/master/travis-ci.sh#L30-L72). It still uses the `e2e` command below. I would like to change that to use the new results command but thought that that should be part of a later change.